metagpt/environment.py

```diff
@@ -41,16 +41,17 @@ def init_roles(self):
         self.add_roles(self.roles.values())
         return self
 
-    def serialize(self, stg_path: Path):
+    def serialize(self, stg_path: Path) -> None:
         roles_path = stg_path.joinpath("roles.json")
-        roles_info = []
+        roles_info: List[Dict[str, Any]] = []
         for role_key, role in self.roles.items():
+            r = self.members.get(role)
```

Comment:
we have a update in dev branch, maybe you can use dev.

Reply:
I was not aware of the used branching model. As there is no documentation on the way you guys develop, nor how to contribute (or if there is any, can you make it clear in the readme, pls?)

Reply:
currently, we update changes in dev and small release-fix in main. So you can merge the changes into dev. And we use pre-commit to check the style.  There still are some `List` `Dict` in the repo, and we will update to `list` `dict`, so just use the new ones.